package.json

```diff
@@ -35,18 +35,18 @@
     "dotenv": "^4.0.0",
     "dotenv-webpack": "^1.5.4",
     "extract-text-webpack-plugin": "^3.0.0",
-    "graphql": "^0.11.0",
-    "graphql-config": "^1.0.0",
+    "graphql": "^0.12.0",
+    "graphql-config": "^1.1.7",
     "html-webpack-plugin": "^2.0.0",
     "postcss-loader": "^2.0.0",
-    "relay-compiler": "^1.0.0",
+    "relay-compiler": "^1.5.0-rc.1",
```

Comment:
It looks some other packages are still lock onto an old relay-compiler. This could be an issue.

```
warning "react-relay > relay-runtime@1.4.1" has incorrect peer dependency "relay-compiler@1.4.1".
```